src/Symfony/Contracts/Service/Attribute/SubscribedService.php

```diff
@@ -19,7 +19,7 @@
  *
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-#[\Attribute(\Attribute::TARGET_METHOD)]
+#[\Attribute(\Attribute::TARGET_METHOD|\Attribute::TARGET_PROPERTY)]
```

Comment:
This makes me realize that ServiceSubscriberInterface is missing a way to convey attributes. Eg how to I use `#[Target]` on a subscribed service? Same for `TaggedIterator` and `TaggedLocator`?

On the declaration side, I think we should just add a second argument to SubscribedService, to allow declaring the attribute:
`#[SubscribedService(attributes: [new Target('foo')])]`

But then, we need to figure out a way to pass that info up to `RegisterServiceSubscribersPass`.

Reply:
Same with the `Autowire` attribute?

Reply:
Yes, same with `Autowire` and `MapDecorated`.

Reply:
Should this be a feature of the `ServiceSubscriberInterface` itself? Then the trait would just configure this based on the `SubscribedService` attributes defined.

Something like:

```php
public static function getSubscribedServices(): array
{
    return [
        new SubscribedService(
            key: 'http_client',
            service: HttpClientInterface::class,
            attributes: [new Target('githubApi')),
    ];
}
```

Reply:
> Something like: [...]

Sounds worth a try. I'd just name the second argument "type" instead of "service", because this is what it is.

Reply:
I was wondering if we couldn't return a closure instead of an array.
The closure would serve only as a signature holder: `return fn (#[Target('bar')] FooInterface $foo) => ;`
That conveys everything and is way nicer for expressing everything.